crates/krun-guest/src/bin/krun-guest.rs

```diff
@@ -48,6 +49,11 @@ fn main() -> Result<()> {
 
     setup_pulse_proxy(run_path)?;
 
+    Command::new("/usr/lib/systemd/systemd-udevd").spawn()?;
```

Comment:
@slp Do we want to go this route?

Reply:
@teohhanhui it's either that or having to implement something equivalent ourselves, which seems an overkill to me. Do you foresee any potential side effects?

Reply:
No, if anything, I'm happy to have `systemd-udevd` 😄

But we should spawn this much earlier so that `dhcpcd` can make use of it, for example. (If that's expected to work...)

Reply:
Should i move this call earlier then?

Reply:
Sure.